contrib/src/main/java/com/datatorrent/contrib/enrich/FSLoader.java

```diff
@@ -145,6 +149,34 @@ private Object getKey(Map<String, Object> tuple)
   @Override
   public Object get(Object key)
   {
+    FSDataInputStream in = null;
+    BufferedReader bin = null;
+    try {
+      this.connect();
```

Comment:
Okay, I added a function where it tries to open a stream with the initialized file system. If it happens to fail it will try to initialize file system once again.